unitOneAssessmentPractice.js

```diff
@@ -3,48 +3,95 @@ let assert = require('assert')
 // Question One:
 
 // Write a function called double that doubles a number
+// const double = (num) =>{
+//   return num *2
+// }
 
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+ //runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
+const containsSeven = (num) =>{
+  num = num.toString()
+  if(num.includes(7)){
```

Comment:
includes already returns a boolean, so you can simply return `num.includes("7")`